baystation12.dme

```diff
@@ -2771,6 +2771,7 @@
 #include "code\unit_tests\zas_tests.dm"
 #include "code\unit_tests\~helpers.dm"
 #include "code\unit_tests\~unit_test_types.dm"
+#include "ds13\code\modules\mob\living\psychosis_new.dm"

```

Comment:
this is a bad code path, don't put ds13 at the start. I'm guessing its an error?

Reply:
Forgot that you weren't staying modular